spec/functions/datetime.md

```diff
@@ -35,93 +31,38 @@ All other _operand_ values produce a _Bad Operand_ error.
 
 ##### Options
 
-The `:datetime` function can use either the appropriate _style options_
-or can use a collection of _field options_ (but not both) to control the formatted
-output.
-_Date/time override options_ can be combined with either _style options_ or _field options_.
-
-If both _style options_ and _field options_ are specified,
-a _Bad Option_ error is emitted
-and a _fallback value_ used as the _resolved value_ of the _expression_.
-
-If the _operand_ of the _expression_ is an implementation-defined date/time type,
-it can include _style options_, _field options_, or other _options_.
-These are included in the resolved option values of the _expression_,
-with _options_ on the _expression_ taking priority over any options of the _operand_.
-
-> [!NOTE]
-> The names of _options_ and their _option values_ were derived from the
-> [options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/resolvedOptions#description)
-> in JavaScript's `Intl.DateTimeFormat`.
-
-###### Style Options
-
-**_<dfn>Style options</dfn>_** pertain to the overall styling or appearance of the formatted output.
-
-The following _style options_ are REQUIRED to be available on the function `:datetime`:
+The following _options_ are REQUIRED to be available on the function `:datetime`:
 
-- `dateStyle`
-  - `full`
-  - `long`
-  - `medium`
-  - `short`
-- `timeStyle`
-  - `full`
-  - `long`
-  - `medium`
-  - `short`
-
-###### Field Options
-
-**_<dfn>Field options</dfn>_** describe which fields to include in the formatted output
-and what format to use for that field.
-
-> [!NOTE]
-> _Field options_ do not have default values because they are only to be used
-> to compose the formatter.
-
-The following _field options_ are REQUIRED to be available on the function `:datetime`:
-
-- `weekday`
-  - `long`
-  - `short`
-  - `narrow`
-- `era`
-  - `long`
-  - `short`
-  - `narrow`
-- `year`
-  - `numeric`
-  - `2-digit`
-- `month`
-  - `numeric`
-  - `2-digit`
+- `dateFields`
+  - `weekday`
+  - `day-weekday`
+  - `month-day`
+  - `month-day-weekday`
+  - `year-month-day` (default)
+  - `year-month-day-weekday`
```

Comment:
Not super happy with mixing camel case and kebab case, but I don't really want the field sets to be in camel case

Reply:
The casing was a major point of contention in the ICU4X TC, which is one reason we ended up landing on `YMD`. (I remember where I was when we had that discussion: I was in Helsinki walking to the Future Frontent speaker dinner)